core/eolearn/tests/test_eodata_io.py

```diff
@@ -44,16 +44,16 @@
 
 @pytest.fixture(name="eopatch")
 def eopatch_fixture():
-    eopatch = EOPatch(bbox=DUMMY_BBOX)
-    mask = np.zeros((3, 3, 2), dtype=np.int16)
-    data = np.zeros((2, 3, 3, 2), dtype=np.int16)
-    eopatch.mask_timeless["mask"] = mask
-    eopatch.data["data"] = data
-    eopatch.timestamps = [datetime.datetime(2017, 1, 1, 10, 4, 7), datetime.datetime(2017, 1, 4, 10, 14, 5)]
```

Comment:
how many timestamps are generated by default? :thinking:

Reply:
5 why most of the test dose not really care about that ...

Reply:
do two or three test runs of this module, then change the config to two timestamps and try it again. If there is no significant difference in speed (so the difference is under let's say 15%) then keep it as is, otherwise we can think if it would be better to switch back to 2.

eodata IO is not a very speedy submodule for testing, but most issues come from the mockS3. Still, don't want to make it slower.

Reply:
no significant difference